packages/build/src/install/missing.js

```diff
@@ -57,6 +60,12 @@ const ensureDir = async function (autoPluginsDir) {
     return
   }
 
+  // If `.netlify` exists but is not a directory, we remove it first
+  const autoPluginsParent = normalize(`${autoPluginsDir}/..`)
+  if (await isFile(autoPluginsParent)) {
+    await pUnlink(autoPluginsParent)
```

Comment:
[sand] Should we print a warning in this case? Not sure if this was a user mistake or it is intended, but in both cases maybe it's better to inform the user

Reply:
Good point! Fixed.